docs/src/code/docs-style-guide.adoc

```diff
@@ -1029,157 +1209,202 @@ When needed, use images to help the reader understand:
+     to the filename based on this format: `image_name_vX_Y.png`. +
+     If the application version differs for LinuxCNC's, use the former one.
+     For example, for a screenshot taken from the Axis UI of LinuxCNC 2.8.4,
+     a valid name is `axis_v2_8_4.png`.
+  ** If you're adding an illustration that doesn't include parts of the
+     user interface, add the release number corresponding to the release
+     the image was added to. +
+     For a PR added to future 2.9.0 version, a valid name for an illustration
+     is `components_diagram_v2_9_0.png`.
+* Place images in a separate directory named `img/` in the same directory
+  where the `.adoc` document that you're working on is located.
 * Consider using PNG images instead of JPEG.
-* Compress GIFs with https://ezgif.com/optimize or similar tool.
+// TODO Propose FOSS alternative solutions to compress GIFs
 * Images should be used (only when necessary) to illustrate the description
-of a process, not to replace it.
-* See also how to link and embed <<videos,videos>> to illustrate the documentation.
+  of a process, not to replace it.
+* See also how to link and embed <<docs:style:videos,videos>> to illustrate
+  the documentation.
+
+=== Insert the image into content
 
-=== Add the image link to content
+// TODO Expand block image insertion infos
+//   https://asciidoc-py.github.io/userguide.html#_block_macros
+//   https://docs.asciidoctor.org/asciidoc/latest/macros/images/
+The AsciiDoc code for including an image in a document is:
 
-The Markdown code for including an image in a document is:
-`![Image description which will be the alt tag](img/document_image_title_vX_Y.png)`
+----
+`image::img/document_image_title_vX_Y.png["Image description which will be the alt tag"]`
+----
 
 The image description is the alt text for the rendered image on the
-documentation site. For accessibility and SEO, use https://webaim.org/techniques/alttext/[descriptions]
+documentation site. +
+For accessibility and SEO, use https://webaim.org/techniques/alttext/[descriptions]
 that:
 
 * Are accurate, succinct, and unique.
 * Don't use *image of* or *graphic of* to describe the image.
 
+// TODO Add inline image insertion infos
+//   https://asciidoc-py.github.io/userguide.html#_inline_macros
+
+[[docs:style:compress]]
 === Compress images
 
-You should always compress any new images you add to the documentation. One
-known tool is https://pngquant.org/[`pngquant`], which is cross-platform and
-open source. Install it by visiting the official website and following the
-instructions for your OS.
+You should always compress any new images you add to the documentation.
+
+One known tool is https://pngquant.org/[`pngquant`], which is cross-platform
+and open source. Install it by visiting the official website and/or following
+the instructions for your OS.
+
+////
+TODO Use GitLab's or provide our images compression helper script
 
 If you use macOS and want all screenshots to be compressed automatically, read
```

Comment:
We won't be using MacOS, will we (I mean, I do, I am now, but I don't think that's relevant to this file)